### ISSUE TITLE
PATH not accepted in OS X ElCap

### ISSUE THREAD

**Issue by mwmahlberg** [joefitzgerald/go-plus#360]
Environment:
* OSX 10.11.3
* bash 4.3.42(1)-release installed via MacPorts
* Atom 1.5.3
* go-plus 3.5.3

Output of Packages->Go Plus->Display Go Information.

```
Go: go1.5.3 darwin/amd64 (@/opt/local/lib/go/bin/go)
GOPATH: /Users/markus/go
Cover Tool: /opt/local/lib/go/pkg/tool/darwin_amd64/cover
Vet Tool: /opt/local/lib/go/pkg/tool/darwin_amd64/vet
Format Tool: /opt/local/lib/go/bin/gofmt
Lint Tool: /Users/markus/go/bin/golint
Gocode Tool: /Users/markus/go/bin/gocode
Godef Tool: /Users/markus/go/bin/godef
Gocode Status: Enabled
Oracle Tool: /Users/markus/go/bin/oracle
Git: /usr/bin/git
PATH: Not Set
Atom: 1.5.3 (darwin x64 15.3.0)
```

However, regardless of how atom is called via the command line, PATH is always reported as `Not Set`. albeit the envar _is_ set, as shown below:

```
Laptop:~ markus$ echo $PATH
/Users/markus/.rvm/gems/ruby-2.2.1/bin:/Users/markus/.rvm/gems/ruby-2.2.1@global/bin:/Users/markus/.rvm/rubies/ruby-2.2.1/bin:/Users/markus/go/bin:/opt/local/lib/go/bin:/opt/local/Library/Frameworks/Python.framework/Versions/2.7/bin/:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/markus/.rvm/bin:/Users/markus/.rvm/bin:/opt/local/bin:/opt/local/sbin
```

I am not sure wether the missing PATH has any impact. However, I noticed above behavior while trying to find out why `oracle` is not working properly using the go-oracle package.

**Comment by joefitzgerald** [comment]
Acknowledged; a if is on its way shortly.

**Comment by atishpatel** [comment]
Hey, is there a quick fix for this? Not being able to catch my stupid errors on save is really messing with my flow. xD

Thanks in advance!

**Comment by joefitzgerald** [comment]
For what it's worth, this does not appear to cause any issues with functionality; it's just annoying.

**Comment by joefitzgerald** [comment]
Fixed in `v3.5.4`.

**Comment by andradei** [comment]
Confirmed, thank you.
